lib/features/login/presentation/view/screens/profile_screen.dart

```diff
@@ -0,0 +1,186 @@
+                            fit: BoxFit.cover,
+                          ),
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              const SizedBox(height: 64),
+              Text(
+                user.usename?.toUpperCase() ?? '',
+                style: TextStyle(
+                    fontSize: 20,
+                    color: Theme.of(context).colorScheme.onSurface,
+                    fontWeight: FontWeight.w600),
+              ),
+              const SizedBox(height: 10),
+              Text(
+                user.emails.first.email,
+                style: TextStyle(
+                    fontSize: 16,
+                    fontWeight: FontWeight.w500,
+                    color:
+                        Theme.of(context).colorScheme.onSurface.withAlpha(160)),
+              ),
+              const SizedBox(height: 24),
+              Expanded(
+                child: ListView(
+                  padding: const EdgeInsets.all(16.0),
+                  children: [
+                    _ItemListWidget(
+                      title: 'Celular:',
+                      content: user.phones.first.phone,
+                      icon: Icons.phone,
+                    ),
+                    _ItemListWidget(
+                      title: 'Estado:',
+                      content: user.addresses.first.state,
+                      icon: Icons.public,
+                    ),
+                    _ItemListWidget(
+                      title: 'Cidade:',
+                      content: user.addresses.first.locality,
+                      icon: Icons.location_city,
+                    ),
+                    _ItemListWidget(
+                      title: 'Endereço:',
+                      content: user.addresses.first.address1,
+                      icon: Icons.location_on,
+                    ),
+                    _ItemListWidget(
+                      title: 'Bairro:',
+                      content: user.addresses.first.neighborhood,
+                      icon: Icons.my_location_outlined,
+                    ),
+                    _ItemListWidget(
+                      title: 'CEP:',
+                      content: user.addresses.first.postalCode,
+                      icon: Icons.location_on,
+                    ),
+                    _ItemListWidget(
+                      title: 'Criado em:',
+                      content: user.createdAt.substring(0, 19),
+                      icon: Icons.login,
+                    ),
+                    _ItemListWidget(
+                      title: 'Atualizado em:',
+                      content: user.updatedAt.substring(0, 19),
+                      icon: Icons.update,
+                    ),
+                  ],
+                ),
+              )
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _ItemListWidget extends StatelessWidget {
```

Comment:
Que tal adicionar esse widget privado em uma `/widgets` ou `/components` para que o mesmo se acessível e testado com mais facilidade - em questão de testes de UI. 
Digo isso, pois para métodos, temos [essa limitação](https://stackoverflow.com/questions/54219835/how-to-test-private-functions-methods-in-flutter), mas não sei se classes também há - imagino (99% de certeza) que sim.

- Apenas sugestão.

Reply:
Faz sentido, acho que classes privadas limitam o teste sim. Obrigado pela sugestão, vou modificar